### PR TITLE
Ido/mbc add version and add better url exception catching better resolution of issue 

### PR DESCRIPTION
**Pull request by IdoWeka**

### CODE REVIEW

**Review by danishpa — APPROVED** [review]

File: install/resources_generator.py [hunk 1]
```diff
@@ -90,6 +90,8 @@ def _send_request(req):
                 return True
             except error.URLError:
                 pass
+            except timeout:
```
Comment (danlweka) [inline install/resources_generator.py:93]: i think it's deprecated
better use `TimeoutError`

**Comment by shazad-weka** [comment]
@IdoWeka or @danishpa can you resolve the conflicts so we can merge the changes. Thanks!

File: topics/MBC_cluster_conversion/mbc_divider_script.py [hunk 2]
```diff
@@ -240,6 +243,17 @@ def wait_for_container_to_be_ready(container_type, sudo):
...
+    get_host_from_hosts_list_cmd = '/bin/sh -c "weka cluster host {} -J"'.format(host_id)
+    logger.info('Waiting for host to deactivate')
+    retries = 60
+    for i in range(retries):
+        host_row = json.loads(run_shell_command(get_host_from_hosts_list_cmd))
+        if host_row[0]['status'] == 'INACTIVE':
+            return
+        sleep(1)
+
+    logger.error('host {} did not reach inactive status'.format(host_id))
```
Comment (danlweka) [inline topics/MBC_cluster_conversion/mbc_divider_script.py:256]: it's only log, don't you want to raise an error here if the host didn't deactivate?

File: topics/MBC_cluster_conversion/mbc_divider_script.py [hunk 3]
```diff
@@ -240,6 +243,17 @@ def wait_for_container_to_be_ready(container_type, sudo):
         if local_status[container_type.container_name()]['status']['internalStatus']['state'] == 'READY':
             break
 
+def wait_for_host_deactivate(host_id):
+    get_host_from_hosts_list_cmd = '/bin/sh -c "weka cluster host {} -J"'.format(host_id)
+    logger.info('Waiting for host to deactivate')
+    retries = 60
+    for i in range(retries):
+        host_row = json.loads(run_shell_command(get_host_from_hosts_list_cmd))
+        if host_row[0]['status'] == 'INACTIVE':
```
Comment (danlweka) [inline topics/MBC_cluster_conversion/mbc_divider_script.py:252]: not quite safe..
maybe something like
```
if 'status'  in host_row[0] and host_row['status'] == 'INACTIVE':
```
or
```
status = host_row.get('status', '')
if status == 'INACTIVE':
```
maybe add some IndexError/ json.JSONDecodeError as well

**Review by danlweka — APPROVED** [review]

**Pull request merged**